.github/workflows/unit-tests.yml

```diff
@@ -30,6 +30,7 @@ jobs:
       run: python -c "import sys; print(sys.version)"
     - name: Install dependencies
       run: |
+        SKLEARN_ALLOW_DEPRECATED_SKLEARN_PACKAGE_INSTALL=True \
```

Comment:
I think you are missing this one `syne_tune/blackbox_repository/requirements.txt`, could you try to see if this can be removed after removing this one?

Reply:
Thanks. I thought I grepped everything

Reply:
That solves it. I am removing the issue. Thanks